### ISSUE TITLE
Disable Colors

### ISSUE THREAD

**Issue by Jesse-Lucas1996** [projen/projen#1725]
Hello! How do I disable colours similar to the --no-colors flag in CDK deploy? I've been trying and It doesn't seem to be working.

**Comment by Chriscbr** [comment]
I don't believe there's a flag for this at the moment. Pull requests are welcome. 🙂 

It looks like this is how the CDK does it right now: 

https://github.com/aws/aws-cdk/blob/977b521c83362e4ea62ec9ce0c6619a0faa6667b/packages/aws-cdk/lib/cli.ts#L233-L236

**Comment by pgollucci** [comment]
I can take this

**Comment by mrgrain** [comment]
Also should support [NO_COLOR](https://no-color.org/)

**Comment by Jesse-Lucas1996** [comment]
is there an update?
